README.md

```diff
@@ -33,51 +33,53 @@ docker run --privileged --rm linuxkit/binfmt:v0.8
 ```
 sudo snap install yq
 ```
-5. Install essentials build packages
+5. Install deno
+```
+sudo snap install deno
+```
+6. Install essentials build packages
 ```
 sudo apt-get install -y build-essential openssl libssl-dev libc6-dev clang libclang-dev ca-certificates
 ```
-6. Install Rust
+7. Install Rust
 ```
 curl https://sh.rustup.rs -sSf | sh
 # Choose nr 1 (default install)
 source $HOME/.cargo/env
 ```
-7. Install toml
-```
-cargo install toml-cli
-```
-8. Build and install appmgr
+8. Build and install embassy-sdk
 ```
-cd ~/ && git clone https://github.com/Start9Labs/embassy-os.git
-cd embassy-os/appmgr/
-cargo install --path=. --features=portable --no-default-features && cd ~/
+cd ~/ && git clone --recursive https://github.com/Start9Labs/embassy-os.git
+cd embassy-os/backend/
+./install-sdk.sh
 ```
-Now you are ready to build your first EmbassyOS service
+Now you are ready to build your Synapse service
 
 ## Cloning
 
-Clone the project locally. Note the submodule link to the original project(s). 
+Clone the Synapse wrapper locally. Note the submodule link to the original project. 
 
 ```
 git clone https://github.com/Start9Labs/synapse-wrapper.git
 cd synapse-wrapper
+git submodule update --init --recursive
 ```
 
 ## Building
 
-To build the project, run the following commands:
+To build the Synapse service, run the following commands:
 
 ```
 make
 ```
 
 ## Installing (on Embassy)
 
-SSH into an Embassy device.
-`scp` the `.s9pk` to any directory from your local machine.
+Secure copy (`scp`) the `synapse.s9pk` to your Embassy and SSH in.
```

Comment:
This is not necessary when using remote install with `embassy-cli`

Reply:
Planing to change it to include embassy-cli and mention the option to side load via web gui